syntax = "proto3";

package proto_router;

import "graph.proto";

message TransportRouter{
  Graph graph = 1;
}